drbd/templates/res_single_vol_v9.j2

```diff
@@ -1,7 +1,9 @@
 resource {{ name }} {
    device    {{ device }};
-   disk      {{ disk }};
    meta-disk {{ meta_disk }};
+{%- if disk is defined %}
```

Comment:
Please, could you test if this doesn't create any issue in deployments other than gcp?

Reply:
Thanks. Only test in gcp so far, i will test on libvirt/azure before merge this.